sdk/python/v2beta1/tensorflow-mnist.py

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[1]:
+
+
+from kubernetes.client import V1PodTemplateSpec
+from kubernetes.client import V1ObjectMeta
+from kubernetes.client import V1PodSpec
+from kubernetes.client import V1Container
+from kubernetes.client import V1ResourceRequirements
+
+
+# In[2]:
+
+
+from mpijob import V1ReplicaSpec
```

Comment:
v2beta1?

Reply:
This is from the common package referenced by v2beta1 but not defined by v2beta1 so follow the version of common pakcage.